<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>541</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QSpinBox" name="MOY">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="TAILLE">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="nbrPoint">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Générer une Map</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Criez les liens</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>la distance moyenne</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>la taille de la carte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre des points</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QSpinBox" name="NBRV">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="VitesseMoyenne">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="tempSimul">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>600</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="interval">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="intervalDon">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Préparer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Générer la simulation des véhicules</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre des véhicules</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vitesse Moyenne</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temps de simulation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>interval</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>intervalle de données</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre des agents</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="NBRA">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="MOVE">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mouvement</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="COMMUNICATION">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Communictaion</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="GEDON">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Génération de données</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="DISSEMINATION">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dissémination</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>liens()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>senario()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createSenario()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generer()</slot>
  <slot>liens()</slot>
  <slot>senario()</slot>
  <slot>createSenario()</slot>
 </slots>
</ui>
